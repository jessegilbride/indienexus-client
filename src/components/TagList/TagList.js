import React, { Component } from 'react';
// import ArtistListContext from '../../contexts/ArtistListContext';
// import ArtistContext from '../../contexts/ArtistContext';
import './TagList.css';

/* NOTE: tags are not yet designed and implemented into the project. they are a future feature. */

export default class TagList extends Component {
  render() {
    return (
      <p>This is a list of tags.</p>
    )
  }
}