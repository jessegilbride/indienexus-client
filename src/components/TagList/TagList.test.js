import React from 'react';
import ReactDOM from 'react-dom';
import { MemoryRouter } from "react-router-dom";
import renderer from 'react-test-renderer';
import TagList from './TagList';

// package.JSON for ignoring a test: 
/* "test": "react-scripts test --testPathIgnorePatterns=<path>" */

describe('TagList component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
    <MemoryRouter><TagList /></MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
      <MemoryRouter><TagList name="TagList" /></MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

});