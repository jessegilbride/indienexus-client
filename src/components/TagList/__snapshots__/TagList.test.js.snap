// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagList component renders the UI as expected 1`] = `
<p>
  This is a list of tags.
</p>
`;
